spark/src/main/scala/io/delta/tables/DeltaColumnBuilder.scala

```diff
@@ -96,6 +99,69 @@ class DeltaColumnBuilder private[tables](
     this
   }
 
+  /**
+   * :: Evolving ::
+   *
+   * Specify a column as an identity column that is always generated by the system, i.e., does not
+   * allow user specified values using a start of 1 and a step of 1.
```

Comment:
Could we perhaps reword this a bit? "does not allow user specified values using a start of 1 and a step of 1" is a lot without punctuation.